vsphere-manager-blueprint-inputs.yaml

```diff
@@ -197,6 +197,9 @@
 
 #nodejs_source_url: ''
 
+# Manager sanity application
+#sanity_app_source_url: ''
```

Comment:
please remove this from all input files.. users shouldn't be able to override this..